hkmc2DiffTests/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -48,6 +49,11 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
     case ReplHost.Result(msg) =>
       if msg.startsWith("Uncaught") then output(s"Failed to load runtime: $msg")
     case r => output(s"Failed to load runtime: $r")
+    if importQQ.isSet then
+      h.execute(s"const $termNme = (await import(\"${termFile}\")).default;") match
+      case ReplHost.Result(msg) =>
+        if msg.startsWith("Uncaught") then output(s"Failed to load runtime: $msg")
+      case r => output(s"Failed to load runtime: $r")
```

Comment:
You copy-pasted the error-handling code and forgot to update references to "runtime". Please instead factor this code (including the `h.execute` above) into a common function with a parameter.

Reply:
I refactored the code with a helper function `importRuntimeModule`.